src/mood.ts

```diff
@@ -126,6 +128,19 @@ export function meatMood(urKels = false, embezzlers = false): Mood {
     }
   }
 
+  if (
+    myClass() === $class`Accordion Thief` &&
+    myLevel() >= 15 &&
+    !(
+      getWorkshed() === $item`warbear LP-ROM burner` ||
+      have($item`warbear LP-ROM burner` || get("questG04Nemesis") === "finished")
+    )
+  ) {
+    if (have($skill`The Ballad of Richie Thingfinder`))
+      useSkill($skill`The Ballad of Richie Thingfinder`);
```

Comment:
We should enforce this through linting, but two-line if statements without braces are bad.